packages/config/config/devices/0x0371/zw187.json

```diff
@@ -107,84 +77,77 @@
 					"value": 5
 				},
 				{
-					"label": "Only send Basic Set 0xFF when magnet is near",
+					"label": "Only send Basic Set 0xff when magnet is near",
 					"value": 6
 				}
 			]
 		},
 		"4[0xff00]": {
-			"label": "Application Layer Retry Max Retries",
-			"description": "Configure max number of retries from 0-5. 0 means disabled. The device supports an application level retry mechanism when the application of the device has detected a transmission error when Basic Set, Sensor Binary Report or Notification Report (Access Control) message is sent but fails to result in an ACK or Supervision Report.",
-			"unsigned": true,
+			"label": "Max Number of Retries",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 5,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"unsigned": true
 		},
 		"4[0x00ff]": {
-			"label": "Application Layer Retry Wait Time",
-			"description": "Valid value 0-255. Unit is 100ms",
-			"unsigned": true,
+			"label": "Retry Wait Time",
 			"valueSize": 2,
+			"unit": "100 ms",
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"unsigned": true
 		},
```

Comment:
Should we mention that this applies to the commands the device sends?

Reply:
Going with transmission. According to the manual it applies to Binary reports and notificaiton reports too so "command" seems weird.